staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -144,7 +144,8 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 		"SSL Certificate Authority file used to secure etcd communication.")
 
 	fs.BoolVar(&s.StorageConfig.Quorum, "etcd-quorum-read", s.StorageConfig.Quorum,
-		"If true, enable quorum read.")
+		"If true, enable quorum read. It defaults to true and is strongly recommended not setting to false.")
+	fs.MarkDeprecated("etcd-quorum-read", "This flag is deprecated and will be removed in future release.")
```

Comment:
How about `This flag is deprecated and the ability to switch off quorum read will be removed in a future release`?

Reply:
Awesome! Done. Thanks!